doc/maintaining/installing/install-from-source.rst

```diff
@@ -295,6 +295,20 @@ installed, we need to install and configure Solr.
 
         JAVA_HOME=/usr/lib/jvm/java-6-openjdk-i386/
 
+   .. note::
+
+    If you get the message ``HTTP ERROR 500... JSP support not configured.``
+    then you will need to install a solr-jetty bug fix. You do this by::
+
+        cd /tmp
+        wget https://launchpad.net/~vshn/+archive/ubuntu/solr/+files/solr-jetty-jsp-fix_1.0.2_all.deb
+        sudo dpkg -i solr-jetty-jsp-fix_1.0.2_all.deb
+        sudo service solr restart
```

Comment:
Shouldn't this be 
```sudo service jetty restart``` ?

Reply:
HTTP ERROR 404
Problem accessing /solr/admin/index.jsp. Reason:

    missing core name in path